src/components/RescueDetails.tsx

```diff
@@ -275,6 +307,14 @@ export default function RescueDetails({id}: { id: string }) {
         }
     }
 
+    function renderSecondVolunteerElements(rescue : BirdAlert) {
+        if (rescue.secondVolunteer && rescue.twoPersonRescue) {
+            return `, ${rescue.secondVolunteer.replace(",", "").split(' ').reverse().join(' ')}`
```

Comment:
is this to display `Last, First` as `First Last`? maybe we should just store it that way if that's how we want to display it everywhere